integrations/BingAds/browser.js

```diff
@@ -58,23 +58,23 @@ class BingAds {
     const { type, properties, event } = rudderElement.message;
     const { category, currency, value, revenue, total } = properties;
     const payload = {
-      ea: type,
-      el: event,
+      event: type,
```

Comment:
Can we add some destination ref page link in here

Reply:
updated